autogpt_platform/backend/backend/api/features/integrations/router.py

```
@@ -371,6 +376,194 @@ async def callback(
+            user_id=user_id,
+            provider=getattr(provider, "value", None) or str(provider),
+            scopes=requested_scopes,
+            expires_in_seconds=initiation.expires_in + 60,
+            state_metadata={
+                "flow_type": "device_code",
+                "device_code": initiation.device_code,
+                "interval": initiation.interval,
+                "user_code": initiation.user_code,
+            },
+        )
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)